.github/workflows/c.yaml

```diff
@@ -52,18 +52,21 @@ jobs:
           { build_type: "Debug", compiler: { c: "gcc-6", cxx: "g++-6" }, build_shared_libraries: false }
         ]
     name: "Ubuntu 18.04 (Build: ${{ matrix.build_type }}, Compilers: ${{ matrix.compiler.c }}/${{ matrix.compiler.cxx }}, Shared Library: ${{ matrix.build_shared_libraries }})"
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
+    container: ubuntu:18.04
```

Comment:
Maybe we could use our existing libsbp-build container for this?

Reply:
Think we just need to upgrade cmake to 3.2x, having some issues with the keys from downloading via kitware